### PR TITLE
add new osci author frankschlehofer-pc

### PR DESCRIPTION
**Pull request by frankschlehofer**
### Description
Adds a new osci author author to the repository
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/frankschlehofer.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: <Frank Schlehofer>
+short_name: Frankie
+twitter: <twitter handle without the @>
+github: <frankschlehofer>
+---
+
+**<Frankie Schlehofer>** <Something about you!>
```
Comment (ashwin-pc) [inline _authors/frankschlehofer.markdown:8]: @frankschlehofer Can you add something bout yourself here

File: _authors/frankschlehofer.markdown [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: <Frank Schlehofer>
+short_name: Frankie
+twitter: <twitter handle without the @>
+github: <frankschlehofer>
```
Comment (ashwin-pc) [inline _authors/frankschlehofer.markdown:5]:
```suggestion
name: Frank Schlehofer
short_name: Frankie
github: frankschlehofer
```

Can you make the following changes?

**Comment by dtaivpp** [comment]
@frankschlehofer can you please do the changes requested above? Thank you!

**Comment by dtaivpp** [comment]
Closing as requested changes were not made and we are past deadlines.

**Pull request closed without merging**